ckanext/geocat/utils/ogdch_map_utils.py

```diff
@@ -206,7 +206,7 @@ def map_resource(geocat_resource, issued, modified, rights):
     resource_dict['issued'] = issued
     resource_dict['modified'] = modified
     resource_dict['rights'] = rights
-    resource_dict['license'] = rights
+    resource_dict['license'] = license
```

Comment:
If we always have `license = rights`, we can skip passing in the license here. :)

Reply:
At least the code changes are minimalistic :-D